src/gui/notificationwidget.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) by Klaas Freitag <freitag@owncloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#ifndef NOTIFICATIONWIDGET_H
+#define NOTIFICATIONWIDGET_H
+
+#include <QWidget>
+
+#include "activitydata.h"
+
+#include "ui_notificationwidget.h"
+
+class QProgressIndicator;
+
+namespace OCC {
+
+class NotificationWidget : public QWidget
+{
+    Q_OBJECT
+public:
+    explicit NotificationWidget(QWidget *parent = 0);
+
+    void setAccountName( const QString& name );
```

Comment:
This function is unnecessary, since setActivity() also sets the account name based on the activity.